synthtool/gcp/templates/python_library/noxfile.py.j2

```diff
@@ -64,16 +64,9 @@ def lint(session):
     session.run("flake8", "google", "tests")
 
 
-@nox.session(python="3.6")
+@nox.session(python=DEFAULT_PYTHON_VERSION)
```

Comment:
What is the current version of DEFAULT_PYTHON_VERSION and where is it set?

Reply:
It is currently 3.8 and is set on line 30.

https://github.com/googleapis/synthtool/blob/b9a6038545c140280f971faa3d43218f7e5eba69/synthtool/gcp/templates/python_library/noxfile.py.j2#L30

https://github.com/googleapis/synthtool/blob/898b38a6f4fab89a76dfb152480bb034a781331b/synthtool/gcp/common.py#L225-L226